setup.py

```diff
@@ -23,7 +23,10 @@
     long_description=readme,
     author="Martin Schrimpf",
     author_email='martin.schrimpf@outlook.com',
-    url='https://github.com/mschrimpf/result_caching',
+    #url='https://github.com/mschrimpf/result_caching',
+
+    # above URL creates organization dependency errors:
```

Comment:
dead code, remove. We can re-trace from the git history

Reply:
result_caching now points to Brain-Score URL instead of Martin's.